packages/clarity-js/types/layout.d.ts

```diff
@@ -87,6 +98,8 @@ export const enum JsonLD {
 
 export const enum Setting {
     LookAhead = 33, // 33ms
+    MutationSuspendThreshold = 10, // Stop listening for mutations after hitting a threshold
+    MutationActivePeriod = 3000 // Let mutations continue as normal during active periods of user interactions
```

Comment:
>3000 [](http://example.com/codeflow?start=27&length=4)

What are the time units for these? ms or seconds?

Reply:
Added it in comments. The answer is milliseconds.